src/error_estimation/smoothness_estimator.C

```diff
@@ -132,19 +132,19 @@ Real SmoothnessEstimator::compute_slope(int N, Real Sx, Real Sy, Real Sxx, Real
     return (N * Sxy - Sx * Sy) / denom;
 }
 
-void SmoothnessEstimator::reduce_smoothness (std::vector<ErrorVectorReal> & error_per_cell,
+void SmoothnessEstimator::reduce_smoothness (std::vector<ErrorVectorReal> & _smoothness_per_cell,
```

Comment:
I definitely like the name change here, except for the leading underscore - that's supposed to be a hint that a variable is a private member, and you don't want it on a function parameter.